search-app/src/cmr/search/services/parameters/parameter_validation.clj

```diff
@@ -687,12 +687,30 @@
                   "The only format that is supported is xml.")
              (name (:result-format params)))]))
 
+(defn- validate-deleted-colls-revision-date-str
+  [revision-date]
+  (when (.contains revision-date ",")
+    [(format (str "Invalid format for revision date, only a starting date is allowed "
+                  "for deleted collections search, but was [%s]") revision-date)]))
+
+(defn- deleted-colls-revision-date-validation
+  "Validates that revision date can only have a start date for deleted collections search"
+  [params]
+  (when-let [revision-date (or (:revision-date params)
+                               (:revision_date params))]
+    (if (sequential? revision-date)
+      (if (> (count revision-date) 1)
+        [(format "Only one revision date is allowed, but was %s" (count revision-date))]
```

Comment:
Maybe something like "Only one revision date is allowed, but %s were provided" (or specified)